### DISCUSSION TITLE
用的vue版本的小程序，chooseAvatar回调无返回

### DISCUSSION THREAD

**Discussion by gumpBOBO** [NervJS/taro#13290] [Q & A]
taro版本："@tarojs/taro": "3.5.7", 用的vue3版本。
微信开发工具库：2.30.0 (最新的)

代码按官网给的写的
<button class="avatar-wrapper" open-type="chooseAvatar" onChooseAvatar={(e) => handleChooseAvatar(e)}>
能弹出选微信头像，但是点击后就是无法触发回调？

**Comment by TheKonka** [comment]
`@ChooseAvatar="handleChooseAvatar" `
这样写看看

**Reply by gumpBOBO** [reply]
我是tsx写的。。。不是模版。。

**Reply by TheKonka** [reply]
```diff
- onChooseAvatar
+ onChooseavatar
```
https://docs.taro.zone/docs/vue-overall#taro-%E8%A7%84%E8%8C%83-1
https://github.com/NervJS/taro/issues/8796
https://github.com/NervJS/taro/issues/11445

**Reply by yoyo837** [reply]
只大写首个单词? 那是不是改一下文档好一点.
